rook/remove-rook.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+namespace="rook-ceph"
+release_name="rook-ceph"
+
+# Remove toolbox
+kubectl --namespace "${namespace}" delete -f "${here}/toolbox-deploy.yaml"
+
+# Remove ceph cluster
+kubectl -n "${namespace}" patch cephclusters.ceph.rook.io "${namespace}" -p '{"metadata":{"finalizers": []}}' --type=merge
```

Comment:
Shouldn't the operator remove the finalizer?

Reply:
It should... but it got stuck if i did not do the path. Found some github issues about it.